File: tests/rspec/pages/members/profile/personal_settings.rb
require_relative '_profile_form.rb'

module Profile
  class PersonalSettings < ControlPanelPage
    set_url '/admin.php?/cp/members/profile/settings'
    section :profile_form,
      Profile::ProfileForm,
      'form[action*="cp/members/profile"]'
  end
end

File: tests/cypress/cypress/elements/pages/members/profile/PersonalSettings.js
import ProfileForm from './ProfileForm'

class PersonalSettings extends ProfileForm {
  constructor() {
      super()
      this.url = '/admin.php?/cp/members/profile/settings'

      this.elements({

        'profile_form': 'form[action*="cp/members/profile"]',
        'profile_form_submit': 'form[action*="cp/members/profile"] div.form-btns.form-btns-top input[type=submit]'
      })
  }
}
export default PersonalSettings;